### DISCUSSION TITLE
Add Power Plan Menu

### DISCUSSION THREAD

**Discussion by Demoe17** [moudey/Shell#401] [Q&A] [answered]
Hello!
I need some help with this software as I am extremely clueless as to how to even go about this and I have been trying and trying but nothing seems to work. Everytime I add someting in the shell.nss file everything breaks and I end up with the classic WIN 10 context menu instead of the one from shell.
I have been looking at the examples and the documentation provided for this software but unfortunately it's not getting me anywhere (I mean literally no where)...
I want to add an option into the context menu so that I can change my windows power plan on the fly via the context menu. Is this even possible and if it is, how would I go about adding that code and where?

**Comment by RubicBG** [comment, marked as answer]
https://discord.com/channels/1106387012707168318/1106387015425069151/1197887784745177139
https://discord.com/channels/1106387012707168318/1106387015425069151/1197892082820452352
there will be an improved version of these two coming soon
